### PR TITLE
Chore/update schedule data

### PR DESCRIPTION
**Pull request by Alperdec**
Problem
=======
What problem or issue does this pull request solve [C2D-188](https://cruzhacks-dev.atlassian.net/browse/C2D-188)

Solution
========
updated schedule to include swag distribution times and name changes to IOS development and UI Design workshops

Change Summary:
---------------
* Tidy, well formulated commit message
* Another great commit message
* Something else I/we did

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
image
image

-----------------------
* Insert any notes/issues, dependencies added or removed, or images

### CODE REVIEW

File: src/Props/schedule/schedule.ts [hunk 1]
```diff
@@ -22,15 +24,18 @@ export const schedule: ScheduleDate[] = [
...
       { name: "Intro to Github", time: "11:00 AM" },
       { name: "What Could Go Wrong", time: "" },
       { name: "Basics of HTML", time: "12:00 PM" },
-      { name: "Intro To UI Design", time: "1:00 PM" },
+      { name: "Intro To UI Design → Intro to UI Design", time: "1:00 PM" },
+      { name: "Swag Distribution Ends", time: "2:00 PM" },
       { name: "Android Development", time: "2:00 PM" },
       { name: "Intro to Python", time: "3:00 PM" },
-      { name: "iOS Development", time: "4:00 PM" },
+      { name: "Intro to IOS development → Intro to React", time: "4:00 PM" },
```
Comment (krishnakpandian) [inline src/Props/schedule/schedule.ts]: Let's remove the first half to only be "Intro to React"

File: src/Props/schedule/schedule.ts [hunk 2]
```diff
@@ -22,15 +24,18 @@ export const schedule: ScheduleDate[] = [
...
     events: [
+      { name: "Pitching for Hackers", time: "9:00 AM" },
       { name: "Intro to Algorithm / Applications", time: "10:00 AM" },
       { name: "Tech Interviews", time: "" },
+      { name: "Swag Distribution Starts", time: "11:00 AM" },
       { name: "Intro to Github", time: "11:00 AM" },
       { name: "What Could Go Wrong", time: "" },
       { name: "Basics of HTML", time: "12:00 PM" },
-      { name: "Intro To UI Design", time: "1:00 PM" },
+      { name: "Intro To UI Design → Intro to UI Design", time: "1:00 PM" },
```
Comment (krishnakpandian) [inline src/Props/schedule/schedule.ts]: Again let's remove the first half to leave it as "Intro to UI Design"

File: src/Props/schedule/schedule.ts [hunk 3]
```diff
@@ -22,15 +24,18 @@ export const schedule: ScheduleDate[] = [
   {
     date: "Saturday, January 15, 2022",
     events: [
+      { name: "Pitching for Hackers", time: "9:00 AM" },
       { name: "Intro to Algorithm / Applications", time: "10:00 AM" },
       { name: "Tech Interviews", time: "" },
+      { name: "Swag Distribution Starts", time: "11:00 AM" },
       { name: "Intro to Github", time: "11:00 AM" },
```
Comment (krishnakpandian) [inline src/Props/schedule/schedule.ts:30]: Let's set the time to blank for consistency

File: src/Props/schedule/schedule.ts [hunk 4]
```diff
@@ -12,6 +12,8 @@ export const schedule: ScheduleDate[] = [
   {
     date: "Friday, January 14, 2022",
     events: [
+      { name: "Swag Distribution Starts", time: "3:00 PM" },
+      { name: "Swag Distribution Ends", time: "5:00 PM" },
       { name: "Event Begins", time: "5:00 PM" },
```
Comment (krishnakpandian) [inline src/Props/schedule/schedule.ts:16]: Let's remove the time here so we have a blank spot

**Review by krishnakpandian — CHANGES_REQUESTED** [review]
Looks Good, just a couple small changes

File: src/Props/schedule/schedule.ts [hunk 5]
```diff
@@ -12,7 +12,9 @@ export const schedule: ScheduleDate[] = [
   {
     date: "Friday, January 14, 2022",
     events: [
+      { name: "Swag Distribution Starts", time: "" },
       { name: "Event Begins", time: "5:00 PM" },
```
Comment (krishnakpandian) [inline src/Props/schedule/schedule.ts]: Last thing, let's switch the times here so the 5PM is at the top.

**Review by krishnakpandian — CHANGES_REQUESTED** [review]
Looks good. I don't know what you think but should we increase the min-height for desktop a little bit for the second page.

File: src/Props/schedule/schedule.ts (hunk 5 above)
Comment (Alperdec) [reply src/Props/schedule/schedule.ts]: the jira ticket says the start time is at 3:00PM are we sure we want to remove the time / move it and imply it starts after 5?

Comment (krishnakpandian) [reply src/Props/schedule/schedule.ts]: Oh my bad, then this should have the time at 3:00PM for Swag Distribution and remain the same. I must have gotten confused initially with the 2 other 5:00 times while reviewing this. My mistake, sorry!

Comment (Alperdec) [reply src/Props/schedule/schedule.ts]: np! just making sure

**Review by krishnakpandian — APPROVED** [review]
LGTM

**Pull request merged**